node/package.json

```diff
@@ -70,8 +72,12 @@
     "//": [
         "The fields below have been commented out and are only necessary for publishing the package."
     ],
-    "///cpu": ["${node_arch}"],
-    "///os": ["${node_os}"],
+    "///cpu": [
```

Comment:
@barshaul this is problematic for the CI, right?

Reply:
sorry, the CD

Reply:
I think it got in by mistake, will remove

Reply:
the lint of it?

Reply:
IIRC this has to stay in a single line.

Reply:
yes, that's true. It cannot be reformatted.

Reply:
adjust npm-cd to fit this structure